.github/workflows/R-CMD-check.yaml

```diff
@@ -31,6 +31,7 @@ jobs:
           - {os: macOS-latest,   pandoc: '2.7.3',   r: 'release'}
           - {os: ubuntu-18.04,   pandoc: '2.7.3',   r: 'oldrel',  rspm: "https://packagemanager.rstudio.com/cran/__linux__/bionic/latest"}
           - {os: ubuntu-18.04,   pandoc: '2.7.3',   r: 'devel',   rspm: "https://packagemanager.rstudio.com/cran/__linux__/bionic/latest", http-user-agent: "R/4.0.0 (ubuntu-16.04) R (4.0.0 x86_64-pc-linux-gnu x86_64 linux-gnu) on GitHub Actions" }
+          - {os: ubuntu-18.04,   pandoc: '2.0.1',   r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/bionic/latest"}
```

Comment:
because 2.0 provides `pandoc-2.0-3-amd64.deb`, which does not match `pandoc-{{version}}-1-amd64.deb`.
However, I just noticed 2.0.0.1 is the minimal version that matches the above format.
Maybe 2.0.0.1 is the choice.

https://github.com/jgm/pandoc/releases?after=2.1

Reply:
Oh I see it is a `r-lib/actions/setup-pandoc` limitation. 

It seems Pandoc 2.0.0 has not the same format. Earlier versions seems to have one. We could also look at other os. 

What is the aim of adding this version ? Testing that the tests are skipped ? 
I was under the impression you suggested to add a CI for Pandoc < 2.0 initially

Reply:
I'm currently testing 2.0 and 2.0.0.1 with GitHub Actions.
Let's employ the minimal install-able version.
61fb7e9

Reply:
Sorry, I missed your latest comment.
About Pandoc < 2.0, I've tested it but raising error which is not related to lua filter.
I think we should fix it and add to test in the different PR.

> Error: Error: pandoc document conversion failed with error 83
> https://github.com/atusy/rmarkdown/pull/2/checks?check_run_id=2219792064